src/test/java/seedu/address/testutil/EditModuleDescriptorBuilder.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.testutil;
+
+import seedu.address.logic.commands.EditCommand.EditModuleDescriptor;
+import seedu.address.model.module.Module;
+import seedu.address.model.module.ModuleName;
+import seedu.address.model.module.ZoomLink;
+
+/**
+ * A utility class to help with building EditModuleDescriptor objects.
+ */
+public class EditModuleDescriptorBuilder {
+
+    private EditModuleDescriptor descriptor;
+
+    public EditModuleDescriptorBuilder() {
+        descriptor = new EditModuleDescriptor();
+    }
+
+    public EditModuleDescriptorBuilder(EditModuleDescriptor descriptor) {
+        descriptor = new EditModuleDescriptor(descriptor);
+    }
+
+    /**
+     * Returns an {@code EditModuleDescriptor} with fields containing {@code module}'s details
```

Comment:
Also not updated for new info in the Module class, will update in the future commits